src/lib/organisms/methodsHeader.svelte

```diff
@@ -1,18 +1,14 @@
 <script>
   export let data;
+  const method = data.methods[0]
```

Comment:
Passing in an array into this component is not logical. It should be a single object. 

Hint: refactor the GraphQL factor first.

Reply:
First there was a single object, but when I added the pdf function it didn't work anymore. I added this yesterday with Justus. Otherwise I will need a little help with this :)

Reply:
Let's look at it together tomorrow!